compiler/src/main/kotlin/walkers/value.kt

```diff
@@ -61,7 +78,9 @@ class LiteralExtractorVisitor : BaseASTVisitor() {
 
         node.ident = node.ctx.neighbourhood_ident().text
 
+        // For each declared coordinate produce a Coordinate through the map operation
         val coords =  node.ctx.neighbourhood_code().coords_decl().map {
```

Comment:
I think you forgot to reformat before shipping. I know the default rules mess with line comments following a statement, and this may be the reason as to why you didn't.
```suggestion
        val coords = node.ctx.neighbourhood_code().coords_decl().map {
```